src/windows/osd.c

```diff
@@ -401,19 +401,29 @@ ssize_t sendmsg(SOCKET sd, struct msghdr *msg, int flags)
 	goto fn_complete;
 }
 
+static inline void getifaddrs_init(int* check_ipv6)
+{
+	fi_param_define(NULL, "use_ipv6", FI_PARAM_BOOL,
+		"Enabling the supporting of IPv6 (default: no)");
+	fi_param_get_bool(NULL, "use_ipv6", &(*check_ipv6));
+}
```

Comment:
Please avoid environment variables for this.  Just add ipv6 support and enable it always.

Reply:
@shefty Cleaned up.